content/docs/started/getting-started-icp.md

```diff
@@ -0,0 +1,193 @@
++++
+title = "ICP For Kubeflow"
```

Comment:
Do not mention ICP, we should always use the long name "IBM Cloud Private"

Reply:
Updated.